### ISSUE TITLE
getSender regex bug

### ISSUE THREAD

**Issue by jeromedecoster** [jankeesvw/LogMeister#5]
Hello,

I just discover a little bug in the 2 regex here:

https://github.com/base42/LogMeister/blob/master/source/logmeister/logmeister/connectors/AbstractConnector.as#L42

You can see here your current regex bug:
http://regexr.com?2tm7o

And here the corrected version:
http://regexr.com?2tm7l

There miss the "." and the ":" match in the first part of each regexp
.:  

The corrected first regexp:
/at\s([a-zA-Z0-9_.:]+)[\/|\]([a-zA-Z0-9_]+)().+:([\d]+)/

The corrected second regexp:
/at\s([a-zA-Z0-9_.:]+)().+[\/|\]([a-zA-Z0-9_]+).as:([\d]+)/

edit : damn, the HTML render badly the regexp !
It's, after the first 9 :
- undersore 
- back slash
- dot
- :

But with new regex, you need to add 2 lines here :
https://github.com/base42/LogMeister/blob/master/source/logmeister/logmeister/connectors/AbstractConnector.as#L48

```
 var className : String = senderDetails[1];
 var idx:int = className.indexOf("::");
 if (idx > -1) className = className.substr(idx + 2);
```

Not fully tested. Hope that helps. I love LogMeister ;)
Thanks!

Jerome

**Comment by jankeesvw** [comment]
Hi Jerome,

Thanks for the bugreport.

Could you send me a pull request? That would be awesome!!

Cheers,
Jankees

**Comment by jeromedecoster** [comment]
I send you an email here : jankees at base42 dot nl

**Comment by jankeesvw** [comment]
I got the message! Thanks!

**Comment by jankeesvw** [comment]
Fixed in Version 1.8.2! Thanks for reporting this bug!
